as2/6.0/modules/ROOT/pages/index.adoc

```diff
@@ -1,100 +1,97 @@
-= AS2 Connector - Get Started - Mule 4
+= AS2 Connector 6.0 - Mule 4
 :page-aliases: connectors::as2/as2-connectorMule4.adoc
 
 Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Premium]
 
-AS2 Connector Version 6.0
-
 Mule runtime engine (Mule) supports the AS2 protocol through Anypoint Connector for AS2 (AS2 Connector).
 
-The AS2 Connector enables trading partners to transmit and receive AS2 messages with EDI-X12, EDIFACT, XML, or binary payloads. For example:
+AS2 Connector enables trading partners to transmit and receive AS2 messages with EDI-X12, EDIFACT, XML, or binary payloads, for example:
 
-* Send messages, signed or unsigned, over HTTP or HTTPS, following the AS2 protocol
+* Send messages, signed or unsigned, over HTTP or HTTPS, using the AS2 protocol
+* Receive messages over HTTP or HTTPS, using the AS2 protocol
+* Receive asynchronous MDN for sent messages
 
-* Receive messages, also over HTTP or HTTPS, following the AS2 protocol
+For compatibility information and fixed issues, see the AS2 Connector Release Notes.
```

Comment:
```suggestion
```

Reply:
We are keeping this line (it's in the template)--just not linking to the release notes since the link is in the info box